src/components/messages_components/MessageListItem.vue

```diff
@@ -18,7 +18,15 @@
       v-text="text"
     ></v-list-item-subtitle>
     <p v-else v-html="text"></p>
-    <v-list-item-action v-text="message.date"></v-list-item-action>
+    <v-list-item-action v-if="messageDate">
+      <span
+        class="d-flex ga-1 mt-2"
+        :style="{ color: 'gray', 'font-weight': 'bold' }"
+        >{{ $t("news.validity") }}
+        <p :style="{ 'font-weight': 'bold' }">{{ messageDate }}</p></span
+      >
+      <!-- Displaying the formatted date -->
+    </v-list-item-action>
```

Comment:
Shouldn't styling be in the style section? Just wondering...

Reply:
Pleas use the `font-weight-bold`class

Reply:
will do